tests/integration/foglamp/services/core/test_scheduler_get_tasks.py

```diff
@@ -56,9 +56,7 @@
                                        sqlalchemy.Column('reason', sqlalchemy.types.VARCHAR(255)))
```

Comment:
this file should be moved to unit?

Reply:
This will be addressed when dependencies of running foglamp is removed and the test is a pure unit test.